src/PowerShell/Microsoft.WinGet.Configuration.Engine/Commands/ConfigurationCommand.cs

```diff
@@ -74,19 +74,31 @@ public static bool ConfirmConfigurationProcessing(PSCmdlet psCmdlet, bool hasAcc
         /// Open a configuration set.
         /// </summary>
         /// <param name="configFile">Configuration file path.</param>
+        /// <param name="allUsers">If to use all users location.</param>
+        /// <param name="currentUser">If to use current user location.</param>
+        /// <param name="wingetModulePath">If to use the winget module path.</param>
+        /// <param name="customLocation">The custom location path.</param>
         /// <param name="executionPolicy">Execution policy.</param>
         /// <param name="canUseTelemetry">If telemetry can be used.</param>
-        public void Get(string configFile, ExecutionPolicy executionPolicy, bool canUseTelemetry)
+        public void Get(
+            string configFile,
+            bool allUsers,

```

Comment:
Couldn't we have a type that holds all of this information?
```
class ModuleStorage
{
enum Location { CU, AU, WG, Custom };

Location StorageLocation{get;set;} = WG;

String? CustomPath = null;
}
```

Reply:
Can types flow into the Engine?  Could be defined in the command side if so.  Or you can just say "this is part of the majesty of ALC design patterns..." 🤮

Reply:
i changed it to be -ModulePath to be the same as winget, so we don't need it